app/assets/stylesheets/active_admin.scss

```diff
@@ -11,7 +11,7 @@
 // Active Admin's got SASS!
 @import "active_admin/mixins";
 @import "active_admin/base";
-$skinActiveColor: #001CEE;
+$skinActiveColor: #001CeE;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected "#001CeE" to be "#001cee"   color-hex-case